llvm/docs/RISCVUsage.rst

```diff
@@ -129,6 +129,7 @@ on support follow.
      ``Smcdeleg``      Supported
      ``Smcsrind``      Supported
      ``Smepmp``        Supported
+     ``Smrnmi``        Supported
```

Comment:
This should probably be "Assembly Support" - some of the other `*ret` instructions have support in clang for `__attribute((interrupt(<kind>)))` which doesn't yet exist for `mnret` - there's no obligation to add this support in the first PR, but this should mean no one expects it has the same level of featureas as the other `*ret` instructions.
```suggestion
     ``Smrnmi``        Assembly Support
```

Reply:
Okay, thanks for your help.